### ISSUE TITLE
config variables

### ISSUE THREAD

**Issue by mb-wali** [fair-data-austria/invenio-rdm-pure#11]
Also for configuration variables - it's better to use a prefix such as ```INVENIO``` or ```PURE``` to all newly added config vars.  open for discussion @utnapischtim .

_Originally posted by @mb-wali in https://github.com/tu-graz-library/invenio-rdm-pure/pull/9#r514189423_

**Comment by utnapischtim** [comment]
do you mean that all configuration variables should start with the same prefix?

for me it would be the best to distinguish between configuration variables who belong to the repository prefixed by INVENIO and those which belong to the pure system prefixed with PURE.

so we have the following configuration variables 
  - RDM_HOST_URL = "https://127.0.0.1:5000/"
  - RDM_RECORD_URL = RDM_HOST_URL + "api/records/{}"
  - RDM_RECORDS_URL = RDM_HOST_URL + "api/records"
renamed to
  - INVENIO_HOST_URL
  - INVENIO_RECORD_URL
  - INVENIO_RECORDS_URL

and the following configuration variables

  - PURE_API_URL = ""
  - PURE_API_KEY = ""
  - PURE_USERNAME = ""
  - PURE_PASSWORD = ""
  - PURE_RESPONSIBLE_EMAIL = ""

should remain as they are.

SIDE NOTE:

The INVENIO_HOST_URL and the other 2 mentioned above are not more longer here. After the switch to the internal api they should be removed. Therefore i would not put that much effort in those things.

BUT nevertheless in my opinion we should distinguish between the repository part and the target system

further there are the following configuration variables:
  - RDM_PURE_USER_EMAIL = ""  -> INVENIO_PURE_USE_EMAIL
  - RDM_PURE_USER_PASSWORD = ""  -> INVENIO_PURE_USER_PASSWORD

as i remember correctly those two configuration variables are to select/create the user in the repository which is used to associate the created records by the cli/celery/cronjob. Therefore INVENIO_PURE_ in the prefix.

**Comment by mb-wali** [comment]
Yes - I mean all the config variables should start with the same prefix. Because the module is injecting the config variables with prefixes [SEE HERE](https://github.com/tu-graz-library/invenio-rdm-pure/blob/0888fa020f233c6b9abbf3e14b51c746fd12a4e6/invenio_rdm_pure/ext.py#L40)
